task2.css

```
@@ -0,0 +1,60 @@
+* {
+    font-family: 'Open Sans', sans-serif;
+    text-align:justify;
+}
+
+body{
+    margin: 0% 15%;
+}
+
+h1,h2,h4 {
+    color: green;
+}
+
+a{

```

```suggestion
a {
```
И лучше было бы объединить с правилом описанным выше